optimade/server/main.py

```diff
@@ -159,6 +173,29 @@ def get_single_reference(
     )
 
 
+@app.get(
+    "/links",
+    response_model=Union[LinksResponse, ErrorResponse],
+    response_model_skip_defaults=True,
+    tags=["Links"],
+)
+def get_links(request: Request, params: EntryListingQueryParams = Depends()):
+    for str_param in ["filter", "sort"]:
+        if getattr(params, str_param):
+            setattr(params, str_param, "")
+    for int_param in [
+        "page_offset",
+        "page_page",
+        "page_cursor",
+        "page_above",
+        "page_below",
+    ]:
+        if getattr(params, int_param):
+            setattr(params, int_param, 0)
+    params.page_limit = CONFIG.page_limit
+    return u.get_entries(links, LinksResponse, request, params)
```

Comment:
Looks like you ran into the same problems as me, there must be a better way of doing this!

Reply:
I think there is in `pydantic` v1 maybe? Here's hoping.